docs/apidoc/cpp/apigen.rst

```diff
@@ -1,11 +1,6 @@
 C++ API documentation generation
 ================================
 
-.. warning::
-
-   This functionality is experimental, incomplete and subject to
-   backwards-incompatible changes.
-
```

Comment:
So, we're no longer calling this experimental? IIRC, there's unfinished implementations. I created a [label to more easily track `apigen.cpp` issues at a glance](https://github.com/jbms/sphinx-immaterial/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22feat%3A%20apigen.cpp%22)